torchtune/models/mistral/__init__.py

```diff
@@ -23,8 +23,10 @@
     qlora_mistral_7b,
     qlora_mistral_classifier_7b,
 )
+from ._tokenizer import MistralTokenizer
 
 __all__ = [
+    "MistralTokenizer",
```

Comment:
This is in the `__all__`, but I don't see it updated in the model documentation. Therefore any links to this will not render and users won't be able to know what is going on with these tokenizers. Same applies to all model tokenizers.